packages/flame_oxygen/example/README.md

```diff
@@ -0,0 +1,16 @@
+# example
+
+A new Flutter project.
```

Comment:
We could change this auto generated content for a simple one liner saying this is the example, etc

Reply:
Updated